doc/source/categorical.rst

```diff
@@ -515,6 +515,30 @@ To get a single value `Series` of type ``category`` pass in a list with a single
 
     df.loc[["h"],"cats"]
 
+String and datetime related work
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.17.1
+
+The accessors  ``.dt`` and ``.str`` work if the ``s.cat.categories`` are of an appropriate type:
+
+
+.. ipython:: python
+
+    str_cat = Series(list('aabb')).astype('category')
+    str_cat.str.contains("a")
+
+    date_cat = Series(date_range('1/1/2015', periods=5))
+    date_cat.dt.day
+
+.. note::
+
+    The work is done on the ``categories`` and then a new Series is constructed. If the number of
+    categories is much smaller than the the length of the data or you you do lots of string munching
+    work, it can be faster to convert the original ``Series`` to one of type ``category`` and use
+    ``.str.<method>`` or ``.dt.<property>`` on that.
+
+
```

Comment:
I hope this is better:

```

    The work is done on the ``categories`` and then a new ``Series`` is constructed. This has
    some performance implication if you have a ``Series`` of type string, where lots of elements
    are repeated (i.e. the number of unique elements in the ``Series`` is a lot smaller than the 
    length of the ``Series``). In this case it can be faster to convert the original ``Series`` 
    to one of type ``category`` and use``.str.<method>`` or ``.dt.<property>`` on that.
```